### PR TITLE
Create cva6_reorg branch

### PR DESCRIPTION
**Pull request by MikeOpenHWGroup**
This is the branch that contains the prototype of the new CVA6 repo structure.

### CODE REVIEW

**Comment by MikeOpenHWGroup** [comment]
Hi @zarubaf, I have successfully rebased this PR to get the ECA checks to pass.  Can you review and merge?

This will merge to the new `cva6_reorg` branch, **_not_** the `master` branch.

File: .gitmodules [hunk 1]
```diff
@@ -1,57 +1,57 @@
...
+	path = corev_apu/rv_plic
 	url = https://github.com/pulp-platform/rv_plic.git
-[submodule "fpga/src/apb_timer"]
-	path = fpga/src/apb_timer
+[submodule "corev_apu/fpga/src/apb_timer"]
+	path = corev_apu/fpga/src/apb_timer
 	url = https://github.com/pulp-platform/apb_timer.git
-[submodule "tb/dromajo"]
-	path = tb/dromajo
+[submodule "https:/github.com/kabylkas/dromajo.git"]
```
Comment (zarubaf) [inline .gitmodules:52]: That seems off.

**Review by zarubaf — APPROVED** [review]
LGTM in principle. But the diff is too large to really say something meaningful. As it isn't a merge into master we can clean-up with further commits in the future.

Please have a look at the `.gitmodule` and tell me that it is okay, seems off to me.

File: .gitmodules (hunk 1 above)
Comment (MikeOpenHWGroup) [reply .gitmodules]: It _was_ off.  Good catch, thanks!  Pushed in a fix.

**Pull request merged**